pkg/platform/provider/baremetal/cluster/kubeadm.go

```diff
@@ -198,9 +202,6 @@ func (p *Provider) getAPIServerExtraArgs(c *v1.Cluster) map[string]string {
 		args["authorization-webhook-config-file"] = constants.KubernetesAuthzWebhookConfigFile
 		args["authorization-mode"] = "Node,RBAC,Webhook"
 	}
-	for k, v := range c.Spec.APIServerExtraArgs {
-		args[k] = v
-	}
 
```

Comment:
why remove above lines?

Reply:
it's redundant. the assignment has been done in the next line "utilruntime.Must(mergo.Merge(&args, c.Spec.APIServerExtraArgs))"